### PR TITLE
Add Community Choice 2019: Results newspost

### PR DESCRIPTION
**Pull request by Ephemeralis**

### CODE REVIEW

File: news/2020-02-10-community-choice-2019-results.md [hunk 1]
```diff
@@ -0,0 +1,109 @@
...
+date: 2020-02-10 08:00:00 +0000
+---
+
+You've had your say, and now the votes are in! Check out which beatmaps were the Community Favorites of 2019!
+
+![](/wiki/shared/news/2020-01-23-community-choice-2019-voting-open/header.jpg)
+
+Just to remind you, the top-voted map for each mode will become the Community Favorite of 2019 and earn their mapper (or set host) a shiny badge detailing how much everybody loves their work, plus 6 months of osu!supporter to tie it all off. The top ten finalists of each mode will also receive 3 months of osu!supporter to commemorate their near victory.
+
+Lots of maps to churn through, so let's get straight into the results without delay:
```
Comment (cl8n) [inline news/2020-02-10-community-choice-2019-results.md]:
```suggestion
There's lots of maps to churn through, so let's get straight into the results:
```

File: news/2020-02-10-community-choice-2019-results.md [hunk 2]
```diff
@@ -0,0 +1,109 @@
...
+
+### osu!
+
+[![](/wiki/shared/news/2020-02-10-community-choice-2019-results/ghost-rule.jpg)](https://osu.ppy.sh/beatmapsets/413117)
+
+**1421** voters woke up to **[Awaken](https://osu.ppy.sh/users/1380253)**'s set for **[DECO\*27 - Ghost Rule](https://osu.ppy.sh/beatmapsets/413117)**.
+
+This should come as little surprise to anyone, since the set had been sitting at both the top of the favorites list and the most played beatmaps ranking for a solid quarter of a year.
```
Comment (cl8n) [inline news/2020-02-10-community-choice-2019-results.md]: think this should be the part of the previous paragraph, just to cut down some of this vertical space

same thing happens for catch

File: news/2020-02-10-community-choice-2019-results.md [hunk 3]
```diff
@@ -0,0 +1,109 @@
...
+- **[PSYQUI - Education](https://osu.ppy.sh/beatmapsets/983953)** by **[Dergo](https://osu.ppy.sh/users/7439226)** with **157** votes.
+- **[Camellia - potential curve](https://osu.ppy.sh/beatmapsets/913062)** by **[FAMoss](https://osu.ppy.sh/users/7707789)** with **148** votes.
+- **[a crowd of rebellion - The Crow](https://osu.ppy.sh/beatmapsets/884441)** by **[[ A v a l o n ]](https://osu.ppy.sh/users/4632359)** with **145** votes.
+- **[yuikonnu - Houkago Stride](https://osu.ppy.sh/beatmapsets/703670)** by **[tailsdk](https://osu.ppy.sh/users/6751666)** with **142** votes.
+
+**[View the full voting results here.](https://osu.ppy.sh/community/contests/88)**
+
+---
+
+Thank you to everybody who took the time out of their day to vote! It's never easy to pick out only ten especially when the average osu! player experiences hundreds of maps throughout each year.
```
Comment (cl8n) [inline news/2020-02-10-community-choice-2019-results.md]:
```suggestion
Thank you to everybody who took the time out of their day to vote! It's never easy to pick out only ten, especially when the average osu! player experiences hundreds of maps throughout each year.
```

**Comment by cl8n** [comment]
on second thought the bulleted lists I added are probably better as numbered lists

**Comment by nl-tatatat** [comment]
shouldn't eiri- be credited for the quaver taiko beatmap? She was the one who made the entire taiko portion, not sotarks.

**Pull request merged**